### PR TITLE
Sprint 3

### PR DESCRIPTION
**Pull request by Beilmann**
### Description

Please include a summary of the change and which issue is fixed. Also, please include the motivation and context for the change. List any dependencies that are required for this change.

**Check the relevant boxes:**
- [x] Bug fix (nonbreaking change that fixes an issue)
- [x] New feature (nonbreaking change that adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [x] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [ ] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

**Comment by Beilmann** [comment]
@ocofaigh please review

File: submodules/power_management_services_setup/scripts/squid_proxy.sh [hunk 1]
```diff
@@ -147,8 +181,9 @@ if [ "$OS_DETECTED" == "RHEL" ]; then
      grep -qx "export HTTP_proxy=http://$proxy_ip_and_port" "$FILE"  || echo "export HTTP_proxy=http://$proxy_ip_and_port"  >> "$FILE"
      grep -qx "export HTTPS_proxy=http://$proxy_ip_and_port" "$FILE" || echo "export HTTPS_proxy=http://$proxy_ip_and_port" >> "$FILE"
      grep -qx "export no_proxy=$no_proxy_ip" "$FILE"                 || echo "export no_proxy=$no_proxy_ip"                 >> "$FILE"
-     grep -qx "proxy=http://$proxy_ip_and_port" "$FILE"              || echo "proxy=http://$proxy_ip_and_port"              >> /etc/dnf/dnf.conf
-
+     ## this line is incorrect
+     #grep -qx "proxy=http://$proxy_ip_and_port" "$FILE"              || echo "proxy=http://$proxy_ip_and_port"              >> /etc/dnf/dnf.conf
```
Comment (ocofaigh) [inline submodules/power_management_services_setup/scripts/squid_proxy.sh:185]: remove comment

File: submodules/power_management_services_setup/scripts/squid_proxy.sh [hunk 2]
```diff
@@ -26,28 +26,29 @@ exit_abnormal() {                         # Function: Exit with error.
...
+    exit 1
 fi
 
 while getopts :h?:p:in: flag
 do
     case "${flag}" in
-		p)
-		proxy_ip_and_port=${OPTARG};;
+	  p)
+	    proxy_ip_and_port=${OPTARG};;
```
Comment (ocofaigh) [inline submodules/power_management_services_setup/scripts/squid_proxy.sh:36]: can we be consistent with indentation here? It is a bit of a mess right now:

image

**Review by ocofaigh — CHANGES_REQUESTED** [review]
@Beilmann left some minor comments. Also be aware that we will have to bump the major version here since you are renaming variable names

**Review by ocofaigh — APPROVED** [review]

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**